src/field/pokemon.ts

```diff
@@ -9,7 +9,7 @@ import BattleInfo, {
   PlayerBattleInfo,
   EnemyBattleInfo,
 } from "#app/ui/battle-info";
-import type Move from "#app/data/moves/move";
+import Move from "#app/data/moves/move";
```

Comment:
```suggestion
import type Move from "#app/data/moves/move";
```
That's weird, you didn't change anything relating to this import but the `type` got removed.